west.yml

```diff
@@ -92,7 +92,7 @@ manifest:
       revision: 1080094bc246357c6c35e80e8b2d5d0ce7e0d963
       path: tools/net-tools
     - name: hal_nxp
-      revision: 60fe8c0ae6e24d165cf42770b95c7df22b778386
+      revision: refs/pull/38/head
```

Comment:
This needs to be updated once https://github.com/zephyrproject-rtos/hal_nxp/pull/38 is merged.

Reply:
Updated, thanks.